apps/crn-server/test/data-providers/contentful/analytics.data-provider.test.ts

```diff
@@ -510,11 +569,112 @@ describe('Analytics Data Provider', () => {
                               sys: {
                                 id: 'interest-group-id-1',
                               },
+                              active: true,
+                            },
+                          ],
+                        },
+                      },
+                    },
+                    {
+                      linkedFrom: {
+                        interestGroupsCollection: {
+                          items: [
+                            {
+                              sys: {
+                                id: 'interest-group-id-2',
+                              },
+                              active: true,
+                            },
+                          ],
+                        },
+                      },
+                    },
+                  ],
+                },
+              },
+            },
+          );
+          contentfulGraphqlClientMock.request.mockResolvedValueOnce(
+            contentfulGraphQLResponse,
+          );
+
+          const result = await analyticsDataProvider.fetchTeamLeadership({});
+
+          expect(
+            result.items[0]!.interestGroupPreviousLeadershipRoleCount,
+          ).toBe(2);
+        });
+
+        test('Should return interestGroupPreviousLeadershipRoleCount of 1 when 2 active users are leaders of 2 different interest groups one of which is inactive', async () => {
```

Comment:
The same thing here, if the team is inactive (if we add `inactiveSince` in the fixture), the test still passes

<img width="706" alt="Screenshot 2024-03-29 at 12 54 17" src="https://github.com/yldio/asap-hub/assets/16595804/439edfaa-44e3-499a-9e1c-bc396a924717">

but the `interestGroupPreviousLeadershipRoleCount` should return 2 in this case.